### DISCUSSION TITLE
Oqtane module packages

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1525] [Q&A]
@sbwalker what will be the procedure to deploy modules and themes for Oqtane.  I see that nuget.org usage has changed to oqtane.net.

**Comment by sbwalker** [comment]
@leigh-pointer I will provide some information about these changes in the dev branch shortly. The changes were discussed previously in #732 and  #1277 but it has taken longer than expected to get them implemented. The high level summary is that nuget.org is an excellent platform for hosting packages - but it was never designed to be a specialized registry service for something like Oqtane. So the goal with this change is to introduce a specialized Oqtane registry service ( which still relies on nuget.org for downloads ). Developers will be able to manage their own product information - and I will certainly need your help with testing based on the fact that you have already created a number of StudioElf modules :)

**Reply by leigh-pointer** [reply]
@sbwalker 
Indeed i get it! Sounds good and happy to help!
